src/compiler/AST/Frontend.elm

```diff
@@ -58,14 +64,14 @@ lambda arguments body =
 
    | Let
        { varName : VarName
-       , varBody : Expr
-       , body : Expr
+       , varBody : LocatedExpr
+       , body : LocatedExpr
        }
-   | Fixpoint Expr
+   | Fixpoint LocatedExpr
    | Operator
        { opName : VarName
-       , left : Expr
-       , right : Expr
+       , left : LocatedExpr
+       , right : LocatedExpr
```

Comment:
I know this is unrelated to your PR but can we please remove this comment?

Reply:
upcoming.